python/python/glide/async_commands/core.py

```diff
@@ -329,6 +329,19 @@ class InsertPosition(Enum):
     AFTER = "AFTER"
 
 
+class FlushMode(Enum):
+    """
+    A modifiers to dictate the flushing mode explicitly.
```

Comment:
```suggestion
    A modifier to dictate the flushing mode.
```

Reply:
Please follow existing documentation under FlushMode.java
https://github.com/aws/glide-for-redis/blob/main/java/client/src/main/java/glide/api/models/commands/FlushMode.java